test/NuGet.Core.Tests/NuGet.Protocol.Tests/RegistrationsBaseUrlTypesTests.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using FluentAssertions;
+using Xunit;
+
+namespace NuGet.Protocol.Tests
+{
+    public class RegistrationsBaseUrlTypesTests
+    {
+        [Fact]
+        public void RegistrationsBaseUrlTypes_RegistrationsBaseUrl_Has_Correct_Coresponding_String_Literal()
+        {
+            RegistrationsBaseUrlTypes.RegistrationsBaseUrl.Should().Be("RegistrationsBaseUrl");
+        }
+
+        [Fact]
+        public void RegistrationsBaseUrlTypes_RegistrationsBaseUrl300beta_Has_Correct_Coresponding_String_Literal()
+        {
+            RegistrationsBaseUrlTypes.RegistrationsBaseUrlVersion300beta.Should().Be("RegistrationsBaseUrl/3.0.0-beta");
```

Comment:
This is testing the value of the string, but not testing the code that's actually used directly which is the static array. 

I think all tests should rely on that instead of the static string that's not really used anywhere outside of the RegistrationsBaseUrlTypes class.

Reply:
In an effort to reduce some noise I ended up ditching the extracted class and reverted back to initializing the array in ServiceTypes. I did update the test to check the expected version strings directly from ServiceTypes.RegstrationsBaseUrl.